CHANGELOG.md

```diff
@@ -43,6 +44,23 @@
 - [v0.2.0](#v020)
 - [v0.1.0](#v010)
 
+## [v1.15.0]
+
+> Release date: 2022/09/23
+
+### Added
+
+- Add support to 2.x to 3.x format migration.
+  [#753](https://github.com/Kong/deck/pull/753)
+- Add support to the new Kong `expressions` flavor fields.
+  [#752](https://github.com/Kong/deck/pull/752)
+- Make sure decK runs against Kong Gateway if a non-default
+  `--kong-addr` is provided, even if some lingering Konnect
+  environment variables are present.
+  [#738](https://github.com/Kong/deck/pull/738)
+- Make sure `validate` with `--online` flag fails when running against Konnect.
```

Comment:
`deck validate` with `--online` flag is not currently supported with Konnect cloud. A clear error message is provided when this command is invoked.